src/data/subscription.json

```diff
@@ -1,15 +1,15 @@
 [
```

Comment:
This file must be deleted, because now all the data must be in mongo db compass

Reply:
I forgot about it. It's deleted now!